CMakeLists.txt

```diff
@@ -137,25 +136,25 @@ IF (DOWNLOAD_TITLE_SEQUENCES)
 endif ()
 if (WIN32)
-    install(TARGETS "libopenrct2" RUNTIME DESTINATION "bin")
+    install(TARGETS "libopenrct2" RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
 else ()
     if (PORTABLE)
-        install(TARGETS "libopenrct2" LIBRARY DESTINATION "bin")
+        install(TARGETS "libopenrct2" LIBRARY DESTINATION "${CMAKE_INSTALL_BINDIR}")
     else ()
-        install(TARGETS "libopenrct2" LIBRARY DESTINATION "lib${LIB_SUFFIX}")
+        install(TARGETS "libopenrct2" LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}")
     endif ()
 endif ()
-install(TARGETS "openrct2" RUNTIME DESTINATION "bin")
-install(TARGETS "openrct2-cli" OPTIONAL RUNTIME DESTINATION "bin")
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/g2.dat" DESTINATION "share/openrct2")
-install(DIRECTORY "data/" DESTINATION "share/openrct2")
-install(FILES ${DOC_FILES} DESTINATION "share/doc/openrct2")
-install(FILES "resources/logo/icon_x16.png" DESTINATION "share/icons/hicolor/16x16/apps" RENAME "openrct2.png")
-install(FILES "resources/logo/icon_x32.png" DESTINATION "share/icons/hicolor/32x32/apps" RENAME "openrct2.png")
-install(FILES "resources/logo/icon_x64.png" DESTINATION "share/icons/hicolor/64x64/apps" RENAME "openrct2.png")
-install(FILES "resources/logo/icon_x128.png" DESTINATION "share/icons/hicolor/128x128/apps" RENAME "openrct2.png")
-install(FILES "resources/logo/icon_x256.png" DESTINATION "share/icons/hicolor/256x256/apps" RENAME "openrct2.png")
-install(FILES "resources/logo/icon_flag.svg" DESTINATION "share/icons/hicolor/scalable/apps" RENAME "openrct2.svg")
-install(FILES "distribution/linux/openrct2.desktop" DESTINATION "share/applications")
-install(DIRECTORY "distribution/man/" DESTINATION "share/man/man6" FILES_MATCHING PATTERN "*.6")
+install(TARGETS "openrct2" RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
+install(TARGETS "openrct2-cli" OPTIONAL RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/g2.dat" DESTINATION "${CMAKE_INSTALL_DATADIR}/openrct2")
+install(DIRECTORY "data/" DESTINATION "${CMAKE_INSTALL_DATADIR}/openrct2")
+install(FILES ${DOC_FILES} DESTINATION "${CMAKE_INSTALL_DOCDIR}/openrct2")
+install(FILES "resources/logo/icon_x16.png" DESTINATION "${CMAKE_INSTALL_DATADIR}/icons/hicolor/16x16/apps" RENAME "openrct2.png")
+install(FILES "resources/logo/icon_x32.png" DESTINATION "${CMAKE_INSTALL_DATADIR}/icons/hicolor/32x32/apps" RENAME "openrct2.png")
+install(FILES "resources/logo/icon_x64.png" DESTINATION "${CMAKE_INSTALL_DATADIR}/icons/hicolor/64x64/apps" RENAME "openrct2.png")
+install(FILES "resources/logo/icon_x128.png" DESTINATION "${CMAKE_INSTALL_DATADIR}/icons/hicolor/128x128/apps" RENAME "openrct2.png")
+install(FILES "resources/logo/icon_x256.png" DESTINATION "${CMAKE_INSTALL_DATADIR}/icons/hicolor/256x256/apps" RENAME "openrct2.png")
+install(FILES "resources/logo/icon_flag.svg" DESTINATION "${CMAKE_INSTALL_DATADIR}/icons/hicolor/scalable/apps" RENAME "openrct2.svg")
```

Comment:
in general this needs to be separated into `CMAKE_INSTALL_DATADIR` and `CMAKE_INSTALL_DATAROOTDIR` because some distros (Debian, formerly Gentoo too) require package-specific data files to be separated out. That is, texture files and stuff consumed only be the package should go into `CMAKE_INSTALL_DATADIR` and stuff that the OS interfaces with (like desktop icons and other FDO dirs) should use `CMAKE_INSTALL_DATAROOTDIR`.